build.gradle

```diff
@@ -237,6 +237,9 @@ allprojects {
         mavenCentral()
         // org.jetbrains.trove4j:trove4j:20160824.
         maven { url 'https://plugins.gradle.org/m2/' }
+        maven {
+            url 'https://trouterpublicpackages.z13.web.core.windows.net'
```

Comment:
We talked about publishing Trouter into maven. If it is published to Maven, we should not add this.

Reply:
Jiri is still working on it, so we have to use this url for now. We will update it once we have the package on maven.